mlflow/genai/datasets/__init__.py

```diff
@@ -0,0 +1,29 @@
+"""
+Databricks Agent Datasets Python SDK. For more details see Databricks Agent Evaluation:
+ <https://docs.databricks.com/en/generative-ai/agent-evaluation/index.html>
+
+The API docs can be found here:
+<https://api-docs.databricks.com/python/databricks-agents/latest/databricks_agent_eval.html#datasets>
+"""
+
+try:
+    from databricks.agents.datasets import (
+        Dataset as EvaluationDataset,
```

Comment:
@smoorjani In what version of the databricks-agents SDK will this class inherit from `mlflow.data.Dataset`, instead of `mlflow.entities.Dataset`?

Reply:
it'll be in the next rc/version of `databricks-agents` so either `1.0.0rc1` or `1.0.0`

Reply:
Thanks!